tests/stubtest_stdlib.py

```diff
@@ -6,55 +6,105 @@
 In typeshed CI, we run stubtest with each currently supported Python minor version.
 
 """
+from __future__ import annotations
 
 import subprocess
 import sys
+import tempfile
 from pathlib import Path
 
+from packaging.requirements import Requirement
+
+from utils import make_venv, print_error
+
 
 def run_stubtest(typeshed_dir: Path) -> int:
     allowlist_dir = typeshed_dir / "tests" / "stubtest_allowlists"
     version_allowlist = f"py{sys.version_info.major}{sys.version_info.minor}.txt"
     platform_allowlist = f"{sys.platform}.txt"
     combined_allowlist = f"{sys.platform}-py{sys.version_info.major}{sys.version_info.minor}.txt"
+    with tempfile.TemporaryDirectory() as tmp:
+        venv_dir = Path(tmp)
+        try:
+            pip_exe, python_exe = make_venv(venv_dir)
+        except Exception:
+            print_error("fail")
+            raise
+
+        # Install the same mypy version as in "requirements-tests.txt"
+        with Path("requirements-tests.txt").open() as requirements_txt:
+            mypy_requirement = next(
+                (
+                    Requirement(requirement.split("#")[0])
+                    for requirement in requirements_txt.readlines()
+                    if requirement.startswith("mypy")
+                )
+            )
```

Comment:
On second thought, we should probably use this preexisting function for this logic, no?

https://github.com/python/typeshed/blob/d92332d120ed0a042d470d326809f9c548be7967/tests/utils.py#L82-L85

(If you want to improve that function in this PR by stripping comments, that's fine!)

Reply:
Didn't see or remember that. Absolutely!